Darkly/LDUserModel.m

```diff
@@ -31,7 +31,7 @@ -(NSDictionary *)dictionaryValue{
     NSMutableDictionary *dictionary = [[NSMutableDictionary alloc] init];
     
     NSDateFormatter *formatter = [[NSDateFormatter alloc] init];
-    [formatter setDateFormat:@"yyyy-MM-ddTHH.mm.ss"];
+    [formatter setDateFormat:@"yyyy-MM-dd'T'HH:mm:ss.SSS'Z'"];
```

Comment:
how come we have to change this? I.e. I think i pulled this from the old code, was this not correct?